WinSW CI Failure: Missing Packages

Azure DevOps currently fails to build the project. Build fails due to the missing dependencies in Microsoft Visual Studio Offline Packages .

```
Microsoft (R) Build Engine version 16.6.0+5ff7b0c9e for .NET Core
Copyright (C) Microsoft Corporation. All rights reserved.

  Determining projects to restore...
D:\a\1\s\src\Test\winswTests\winswTests.csproj : warning NU1603: winswTests depends on Microsoft.NET.Test.Sdk (>= 16.4.0) but Microsoft.NET.Test.Sdk 16.4.0 was not found. An approximate best match of Microsoft.NET.Test.Sdk 16.6.0 was resolved. [D:\a\1\s\src\winsw.sln]
D:\a\1\s\src\Test\winswTests\winswTests.csproj : warning NU1603: NUnit 3.12.0 depends on NETStandard.Library (>= 2.0.0) but NETStandard.Library 2.0.0 was not found. An approximate best match of NETStandard.Library 2.0.3 was resolved. [D:\a\1\s\src\winsw.sln]
D:\a\1\s\src\Test\winswTests\winswTests.csproj : error NU1101: Unable to find package log4net. No packages exist with this id in source(s): Microsoft Visual Studio Offline Packages [D:\a\1\s\src\winsw.sln]
D:\a\1\s\src\Test\winswTests\winswTests.csproj : error NU1101: Unable to find package NUnit3TestAdapter. No packages exist with this id in source(s): Microsoft Visual Studio Offline Packages [D:\a\1\s\src\winsw.sln]
D:\a\1\s\src\Test\winswTests\winswTests.csproj : error NU1101: Unable to find package Microsoft.NETFramework.ReferenceAssemblies. No packages exist with this id in source(s): Microsoft Visual Studio Offline Packages [D:\a\1\s\src\winsw.sln]
D:\a\1\s\src\Test\winswTests\winswTests.csproj : error NU1101: Unable to find package ilmerge. No packages exist with this id in source(s): Microsoft Visual Studio Offline Packages [D:\a\1\s\src\winsw.sln]
D:\a\1\s\src\Test\winswTests\winswTests.csproj : error NU1101: Unable to find package SharpZipLib. No packages exist with this id in source(s): Microsoft Visual Studio Offline Packages [D:\a\1\s\src\winsw.sln]
D:\a\1\s\src\Test\winswTests\winswTests.csproj : error NU1101: Unable to find package Microsoft.CodeCoverage. No packages exist with this id in source(s): Microsoft Visual Studio Offline Packages [D:\a\1\s\src\winsw.sln]
D:\a\1\s\src\Test\winswTests\winswTests.csproj : error NU1101: Unable to find package System.ServiceProcess.ServiceController. No packages exist with this id in source(s): Microsoft Visual Studio Offline Packages [D:\a\1\s\src\winsw.sln]
D:\a\1\s\src\Test\winswTests\winswTests.csproj : error NU1101: Unable to find package Microsoft.TestPlatform.TestHost. No packages exist with this id in source(s): Microsoft Visual Studio Offline Packages [D:\a\1\s\src\winsw.sln]
D:\a\1\s\src\Test\winswTests\winswTests.csproj : error NU1101: Unable to find package Microsoft.Win32.Registry. No packages exist with this id in source(s): Microsoft Visual Studio Offline Packages [D:\a\1\s\src\winsw.sln]
D:\a\1\s\src\Test\winswTests\winswTests.csproj : error NU1101: Unable to find package System.Diagnostics.EventLog. No packages exist with this id in source(s): Microsoft Visual Studio Offline Packages [D:\a\1\s\src\winsw.sln]
D:\a\1\s\src\Test\winswTests\winswTests.csproj : error NU1101: Unable to find package System.Management. No packages exist with this id in source(s): Microsoft Visual Studio Offline Packages [D:\a\1\s\src\winsw.sln]
D:\a\1\s\src\Test\winswTests\winswTests.csproj : error NU1102: Unable to find package System.Runtime.Extensions with version (>= 4.3.1) [D:\a\1\s\src\winsw.sln]
D:\a\1\s\src\Test\winswTests\winswTests.csproj : error NU1102:   - Found 2 version(s) in Microsoft Visual Studio Offline Packages [ Nearest version: 4.1.0 ] [D:\a\1\s\src\winsw.sln]
  Failed to restore D:\a\1\s\src\Test\winswTests\winswTests.csproj (in 3.18 sec).
D:\a\1\s\src\Plugins\SharedDirectoryMapper\SharedDirectoryMapper.csproj : error NU1101: Unable to find package log4net. No packages exist with this id in source(s): Microsoft Visual Studio Offline Packages [D:\a\1\s\src\winsw.sln]
D:\a\1\s\src\Plugins\SharedDirectoryMapper\SharedDirectoryMapper.csproj : error NU1101: Unable to find package Microsoft.NETFramework.ReferenceAssemblies. No packages exist with this id in source(s): Microsoft Visual Studio Offline Packages [D:\a\1\s\src\winsw.sln]
D:\a\1\s\src\Plugins\SharedDirectoryMapper\SharedDirectoryMapper.csproj : error NU1101: Unable to find package SharpZipLib. No packages exist with this id in source(s): Microsoft Visual Studio Offline Packages [D:\a\1\s\src\winsw.sln]
D:\a\1\s\src\Plugins\SharedDirectoryMapper\SharedDirectoryMapper.csproj : error NU1101: Unable to find package System.Diagnostics.EventLog. No packages exist with this id in source(s): Microsoft Visual Studio Offline Packages [D:\a\1\s\src\winsw.sln]
D:\a\1\s\src\Plugins\SharedDirectoryMapper\SharedDirectoryMapper.csproj : error NU1101: Unable to find package System.Management. No packages exist with this id in source(s): Microsoft Visual Studio Offline Packages [D:\a\1\s\src\winsw.sln]
D:\a\1\s\src\Plugins\SharedDirectoryMapper\SharedDirectoryMapper.csproj : error NU1102: Unable to find package System.Runtime.Extensions with version (>= 4.3.1) [D:\a\1\s\src\winsw.sln]
D:\a\1\s\src\Plugins\SharedDirectoryMapper\SharedDirectoryMapper.csproj : error NU1102:   - Found 2 version(s) in Microsoft Visual Studio Offline Packages [ Nearest version: 4.1.0 ] [D:\a\1\s\src\winsw.sln]
  Failed to restore D:\a\1\s\src\Plugins\SharedDirectoryMapper\SharedDirectoryMapper.csproj (in 75 ms).
  Restored D:\a\1\s\src\Core\WinSWCore\WinSWCore.csproj (in 47.5 sec).
  Restored D:\a\1\s\src\Plugins\RunawayProcessKiller\RunawayProcessKiller.csproj (in 47.5 sec).
  Restored D:\a\1\s\src\Core\ServiceWrapper\winsw.csproj (in 47.59 sec).
```

--- a/eng/build.yml
+++ b/eng/build.yml
@@ -28,12 +28,6 @@ jobs:
       Release:
         BuildConfiguration: Release
   steps:
-  - task: DotNetCoreCLI@2
-    inputs:
-      command: 'restore'
-      projects: '**/*.csproj'
-      includeNuGetOrg: true
-    displayName: 'Restoring code using dotnet restore'
   - script: |
       dotnet build -c $(BuildConfiguration) src\winsw.sln -p:Version=$(BuildVersion)
       dotnet publish -c $(BuildConfiguration) -f netcoreapp3.1 src\Core\ServiceWrapper\winsw.csproj -p:Version=$(BuildVersion)

--- a/src/NuGet.config
+++ b/src/NuGet.config
@@ -1,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+    <clear />
+    <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
+  </packageSources>
+</configuration>

